### DISCUSSION TITLE
Is there a way to add dynamic default value?

### DISCUSSION THREAD

**Discussion by AlTosterino** [bhch/django-jsonform#168] [Q&A]
Hey!

Let's assume I have schema like this: `{"type: "string", "readOnly": true}`, is there a fast way to dynamically populate every new value? For example with UUID?

**Comment by bhch** [comment]
If I understand correctly, you want to dynamically populate input fields on the browser?

We have a JS API which might be useful for this: https://django-jsonform.readthedocs.io/en/latest/guide/javascript.html

If I'm wrong, please give me a more illustrative example of your problem.
